content/source/docs/enterprise/api/run.html.md

```diff
@@ -202,6 +202,15 @@ Parameter      | Description
 ---------------|------------
 `workspace_id` | The workspace ID to list runs for.
 
+### Query Parameters
+
+This endpoint supports pagination [with standard URL query parameters](./index.html#query-parameters); remember to percent-encode `[` as `%5B` and `]` as `%5D` if your tooling doesn't automatically encode URLs.
+
+Parameter                    | Description
+-----------------------------|------------
+`page[number]`    | **Optional.** If omitted, the endpoint will return the first page.
+`page[size]` | **Optional.** If omitted, the endpoint will return 20 runs per page.
```

Comment:
Nit-pick but these tables are kind of weird. Seems like you could just pull `Description` to the left some and line everything up all nice like.

Reply:
If your editor supports it, Markdown formatting is a treat here. Although @nfagerlund and myself have editor differences, any kind of formatting is good for now.